### PR TITLE
Update terraform to 0.6.16

### PR DESCRIPTION
**Pull request by netflash**

### CODE REVIEW

File: metadata.rb [hunk 1]
```diff
@@ -5,9 +5,9 @@
 license          'Apache 2.0'
 description      'Installs Terraform (terraform.io)'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          '0.4.4'
+version          '0.5.0'
 
-depends          'ark', '~> 1.0.0'
+depends          'ark', '~> 1.1'
```
Comment (haidangwa) [inline metadata.rb]: I've been of the school of thought stating that dependencies on 3rd party libraries should be considered riskier than those developed yourself or within one's close circle of friends. With that, the constraint on the ark dependency should be down to the patch level: `depends 'ark', '~> 1.1.0'`.

Comment (netflash) [inline metadata.rb]: Agree. Gonna update, run test kitchen and then commit.

**Comment by netflash** [comment]
updated

**Comment by strata-alex** [comment]
Could you please update chef supermarket version ? Thanks.

**Comment by haidangwa** [comment]
@strata-alex sorry that took longer than usual. I was caught up in this thing called "work" last few days. I've uploaded it to supermarket now.

**Comment by netflash** [comment]
W00t! Thanks.

**Pull request merged**